h2o-parsers/h2o-parquet-parser/src/main/java/water/parser/parquet/VecParquetReader.java

```diff
@@ -115,7 +128,7 @@ public void close() throws IOException {
   public static byte[] readFooterAsBytes(Vec vec) {
     FSDataInputStream f = null;
     try {
-      f = new FSDataInputStream(new VecDataInputStream(vec));
+      f = (FSDataInputStream) H2O.getPM().openSeekable(vec);
```

Comment:
Let's make make sure the underlying PM implementation used for Parquest is seekable. Ideally introduce a test.

If it's not seekable, we should react accordingly (open a "normal" input stream ?).

Reply:
yes, adding a test is a good idea - will do

> If it's not seekable, we should react accordingly (open a "normal" input stream ?).

I don't see how this could happen other than someone purposefully excluding persist-hdfs from the assembly - this code would fail but there are other places where it wouldn't even load the classes (since there are hard dependencies on that module)

Reply:
This comment is outdated now. Let's resolve it. Tests are always good anway.

Reply:
okay, thank you Pavel